apps/frontend/src/components/global/Snackbar.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <v-snackbar v-if="show" id="info-snackbar" v-model="show" timeout="10000">
```

Comment:
It may be better to display this on top of the page instead of the bottom to match the existing Toasted configuration.